packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -274,7 +277,7 @@ export function FileUploader({
         newFileStatuses[index] = {
           ...status,
           name: value,
-          fileState: !validExtension ? 'error' : null,
+          fileState: !validExtension ? FileState.ERROR : FileState.INIT,
```

Comment:
If the file extension is invalid should we still be adding it to `fileStatuses`?

Reply:
The way I think of it, is that when an extension is invalid, the file cannot be uploaded, and therefore the file status is in an error state. That status is checked elsewhere to determine if the user can click the 'upload' button. So that's why I made the file state as an error.

Reply:
Just for my understanding, do we render files with invalid extensions in the list of files?

Reply:
Yes, they still get rendered and displayed on the on the preview page. They'll have an error next to them.

Reply:
Got it, thanks for confirming